src/Resources/Locales/de_DE.axaml

```diff
@@ -368,10 +368,11 @@
   <x:String x:Key="Text.Period.LastYear" xml:space="preserve">Leztes Jahr</x:String>
   <x:String x:Key="Text.Period.YearsAgo" xml:space="preserve">Vor {0} Jahren</x:String>
   <x:String x:Key="Text.Preference" xml:space="preserve">Einstellungen</x:String>
-  <x:String x:Key="Text.Preference.Appearance" xml:space="preserve">OBERFLÄCHE</x:String>
+  <x:String x:Key="Text.Preference.Appearance" xml:space="preserve">Erscheinungsbild</x:String>
```

Comment:
"Erscheinungsbild" sounds a little bit archaic to me.
"Aussehen" on the other hand is maybe too colloquial and unusual.
"Design" would be nice word if it wasn't already taken for the selection of "Light"/"Dark" themes.
"Darstellung" may also be another word (used for example in YouTube Android app).
Or - inspired by Windows - "Personalisierung", but this would maybe be too far off the original wording.

Reply:
Wow that's a lot of options. I fully agree tho, so best version to seems like "Darstellung".